win32.mak

```diff
@@ -1,10 +1,6 @@
 MAKE=make
 
 auto-tester-build:
-	cd ../druntime
-	git fetch https://github.com/rainers/druntime.git data_ptrref
-	git merge FETCH_HEAD
-	cd ../dmd
```

Comment:
To get some test coverage for the pointer references. I wanted to remove it before merging (PR said [WIP] in the title), but Walter hit the merge button unexpectedly. I also expected that ELF/macho-o implementations would need to be added before inclusion.